src/hacbs/components/Commits/AppRecentCommits.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Button, PageSection, PageSectionVariants, Text, Title } from '@patternfly/react-core';
+import { pipelineRunstoCommits } from '../../utils/commits-utils';
+import { pipelineWithCommits } from './__mocks__/pipeline-with-commits';
+import CommitsListView from './CommitsListView';
+
+const COMMIT_LIMIT = 3;
+
+const AppRecentCommits = ({ applicationName }) => {
+  const commits = pipelineRunstoCommits(pipelineWithCommits, COMMIT_LIMIT);
+  const navigate = useNavigate();
+  return (
+    <PageSection padding={{ default: 'noPadding' }} variant={PageSectionVariants.light} isFilled>
+      <Title size="lg" headingLevel="h3" className="pf-c-title pf-u-mt-lg pf-u-mb-sm">
+        Recent commits
+      </Title>
+      <Text className="pf-u-mb-lg">
+        Monitor your commits and their pipeline progression across all components
+        <Button variant="link" className="pf-u-pl-sm">
+          learn more
+        </Button>
+      </Text>
+      <CommitsListView commits={commits} />
```

Comment:
we can avoid this duplication by separating it out to a new abstract component that accepts props for title and action buttons and use it in `commitsTab` and in here?

Reply:
updated accordingly